File: Iniciante/1006 - Média 2/Python/1006.py
def main():
    a = float(input())
    b = float(input())
    c = float(input())
    WEIGHT_A = 2
    WEIGHT_B = 3
    WEIGHT_C = 5
    TOTAL_WEIGHT = WEIGHT_A + WEIGHT_B + WEIGHT_C
    student_average = ((a * WEIGHT_A) + (b * WEIGHT_B) + (c * WEIGHT_C)) / TOTAL_WEIGHT
    print(f"MEDIA = {student_average:.1f}")


if __name__ == "__main__":
    main()

File: Iniciante/1006 - Média 2/JavaScript/1006.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const lines = input.split("\n");
  const [a, b, c] = lines.map(Number);
  const WEIGHT_A = 2;
  const WEIGHT_B = 3;
  const WEIGHT_C = 5;
  const TOTAL_WEIGHT = WEIGHT_A + WEIGHT_B + WEIGHT_C;
  const studentAverage = ((a * WEIGHT_A) + (b * WEIGHT_B) + (c * WEIGHT_C)) / TOTAL_WEIGHT;
  console.log(`MEDIA = ${studentAverage.toFixed(1)}`);
}

main();
